File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/colorselector/HueSelectionListener.java
package net.kdt.pojavlaunch.colorselector;

/**
 * 色相が選択されたときのリスナーインターフェース。
 */
public interface HueSelectionListener {
    /**
     * 色相が選択されたときに呼び出されます。
     * @param hue 選択された色相値（0〜360）
     */
   void onHueSelected(float hue);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/ui/view/colorselector/HueSelectionListener.kt
package com.arata.yukarilauncher.ui.view.colorselector

/** 色相が選択された際のコールバックインターフェース。 */
interface HueSelectionListener {
    /** 色相が選択されたときに呼び出される。 @param hue 選択された色相値（0～360） */
    fun onHueSelected(hue: Float)
}